src/flags/sorting.rs

```diff
@@ -426,31 +397,34 @@ mod test_sort_order {
         assert_eq!(None, SortOrder::from_config(&Config::with_none()));
     }
 
-    #[test]
-    fn test_from_config_empty() {
-        let yaml_string = "---";
-        let yaml = YamlLoader::load_from_str(yaml_string).unwrap()[0].clone();
-        assert_eq!(None, SortOrder::from_config(&Config::with_yaml(yaml)));
-    }
-
     #[test]
     fn test_from_config_default() {
-        let yaml_string = "sorting:\n  reverse: false";
-        let yaml = YamlLoader::load_from_str(yaml_string).unwrap()[0].clone();
         assert_eq!(
             Some(SortOrder::Default),
-            SortOrder::from_config(&Config::with_yaml(yaml))
+            SortOrder::from_config(&Config::default())
         );
     }
 
+    #[test]
+    fn test_from_config_none_reverse() {
```

Comment:
Change the test name back to `test_from_config_empty`. It is more consistent and essentially testing the same thing as the previous implementation.